testproject/tests/test_second_step_authentication.py

```diff
@@ -540,21 +495,20 @@ def test_change_primary_method_to_inactive(active_user_with_email_otp):
     response = client.post(
         path="/auth/mfa/change-primary-method/",
         data={
-            "method": "sms",
-            "code": create_otp_code(first_primary_method.secret),
+            "method": "sms_twilio",
+            "code": handler.create_code(),
         },
         format="json",
     )
-    error_code = "missing_method"
     assert response.status_code == 400
```

Comment:
in other tests we are using `rest_framework.status` const so we can do in this test too